### DISCUSSION TITLE
Where are the generated images saved?

### DISCUSSION THREAD

**Discussion by Toroidal-meow** [new-sankaku/stable-diffusion-webui-simple-manga-maker#40] [General]
Hey, great job on making the app!

I was trying it out for a bit, and I was wondering where the images are saved when you generate them with A1111. They don't seem to be saved in the usual Outputs folder.

**Comment by new-sankaku** [comment]
There was a save flag so I turned it ON.
From now on, it will be saved in the same place as normal folders!
